documentation/swagger.yml

```
@@ -2439,6 +2435,29 @@ components:
               - facebook
               - title
               - beacon
+    draft:
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+          description: Draft ID
+        data:
+          $ref: "#/components/schemas/disruption_draft"
+        self:
+          $ref: "#/components/schemas/href"
+        type:
+          type: string
+          enum:
+            - disruption
+        created_at:
+          type: string
+          pattern: '^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}Z$'
+          example: '2021-12-21T00:00:00Z'
```

```suggestion
          example: '2018-08-03T12:19:13Z'
```
more understandable with every data content dates in 2018

+ line 2545 > comming when...  = coming when...